src/pages/Posts.js

```diff
@@ -13,25 +13,43 @@ const Posts = () => {
       .then((data) => setPosts(data));
   }, []);
 
-  const selectedPost = posts.filter((item) => {
-    if (loggedInUser.id === item.userId) {
-      return item;
+
+  const allUsersAndPosts= [
+    {
+      name: "",
+      post: "",
+      title: "",
+    },
+  ];
+  for (let i = 0; i < allUser.length; i++) {
+    for (let k = 0; k < posts.length; k++) {
+      if (allUser[i].id === posts[k].userId) {
+        allUsersAndPosts.push({
+          name: allUser[i].name,
+          post: posts[k].body,
+          title: posts[k].title,
+        });
+      }
     }
-  });
+  }
+
+  const card = allUsersAndPosts.map((randomUser) => {
+    randomUser = allUsersAndPosts[Math.floor(Math.random() * allUsersAndPosts.length)];
 
-  const card = selectedPost.map((item) => {
     return (
       <Card
-        title={item.title}
-        username={loggedInUser.username}
-        postBody={item.body}
+        title={randomUser.title}
+        username={randomUser.name}
+        postBody={randomUser.post}
       />
     );
   });
 
   return (
     <div>
-      <Header />
+      <header>
+        <Header />
+      </header>
```

Comment:
Why wrap a header component in a Header?

Reply:
removed header tags